### DISCUSSION TITLE
Noobie Needs Help

### DISCUSSION THREAD

**Discussion by ProfessorDyson** [Ysurac/openmptcprouter#3503] [Q&A]
Hello all, I’ve recently started getting into OMR. I’ve done quite a bit of research and for someone with 0 background in networking and my only experience with electronics is playing video games, I feel like I’ve made a big jump into the deep end. 

My set is as follows:
-Main PC running Putty and used to access the vps 
(Cloudzy) server and routers.

-Nighthawk M6 Pro (FirstNet)

-InHands (FWA02) multi carrier hotspot through TravlFi

-RPI5 for OMR with an unmanaged gigabyte switch

-Possibly a router to have the bonded internet output as a WiFi signal? (Have not made it to this step)

Where I am at:
I have the VPS install completed on Cloudzy server, RPI5 is accessible at 192.168.100.1 after router image install. Both the M6 Pro and FWA02 as well as my PC are hardwired into the switch which is plugged into the RPI5 Ethernet port. 

So we are all wired together, both the VPS and Router image are installed and seem to be running fine. 

The problem:
When I run the setup wizard inside OMR it kicks me offline and I can no longer access the RPI5 at 192.168.100.1. I’m not sure what I’m doing wrong.

At https://github.com/Ysurac/openmptcprouter/wiki/Router-install
it says that the the modems can be plugged into a switch without the need to purchase usb to Ethernet adapters. Another resource said that if multiple connections are hardwired I would not need to set the modems to a static IP address. Is that accurate? I think that might be where I’m going wrong, but I’m not sure. 

I would need to find the resource again, but it claims if multiple modems are hardwired in then there is no need for static IPs you can use DHCP. But again I’m thinking that this may be the problem. 

Another potential issue is I’m using an unmanaged NetGear gigabyte switch, not sure if a managed switch is required for this application.

I’m not sure where to look to fix my problem, any advice or knowledge would be a huge help.

**Comment by Ysurac** [comment]
If you use DHCP for modems on a RPI5, so only with one ethernet port, with an unmanaged switch, then this can't work.
Set static IPs on modems.

**Reply by ProfessorDyson** [reply]
I had a feeling this was my problem, the resource I was basing my project on was from an article I found on google, it had a couple of other issues too mainly mislabeling. Probably a translation error. 

Glad the King of OMR could clear it up for me. I will try static IPs this afternoon, hopefully it will fix my problem.

**Reply by ProfessorDyson** [reply]
If im reading correctly, I will not be able to use a static IP on my Nighthawk M6 Pro. If I have a managed switch or something like a BPI3 that can accept multi lan in will using DHCP modems work?

**Reply by Ysurac** [reply]
BPI3 is not supported, so no.
For a managed switch you then need to set VLANs.
You can still also add one USB ethernet key on the RPI ofr the Nighthawk.
